app/components/build/AvatarWithUnknownEmailPrompt.js

```diff
@@ -114,49 +117,90 @@ class AvatarWithUnknownEmailPrompt extends React.PureComponent {
       return {};
     }
 
+    const { email: authorEmail } = this.props.build.createdBy;
```

Comment:
Not really - initially I’d called the var `email` and I like the deconstruction syntax for that (vs. repeated “email” on the same line) but it’s a matter of taste, really